repos/fdbt-site/src/pages/api/multiTapsPricing.ts

```diff
@@ -0,0 +1,53 @@
+import { NextApiResponse } from 'next';
+import { MULTI_TAPS_PRICING_ATTRIBUTE } from '../../constants/attributes';
+import { ErrorInfo, MultiTap, MultiTapPricing, NextApiRequestWithSession } from '../../interfaces';
+import { updateSessionAttribute } from '../../utils/sessions';
+import { redirectTo, redirectToError } from '../../utils/apiUtils';
+import { checkPriceIsValid, removeExcessWhiteSpace } from '../../utils/apiUtils/validator';
+
+export const checkAllValidation = (taps: MultiTapPricing): ErrorInfo[] => {
+    const errors: ErrorInfo[] = [];
+
+    Object.entries(taps.tapDetails).forEach((entry) => {
+        const tapPrice = entry[1];
+        const index = entry[0];
+        const trimmedPrice = removeExcessWhiteSpace(tapPrice);
+        const tapPriceError = checkPriceIsValid(trimmedPrice, 'cap', true);
```

Comment:
'cap' should not be passed here, as this error message 'Cap prices cannot be empty' is not appropriate, because we are making the user price their taps, not the cap. 

We should use 'tap'.